packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -1838,6 +1981,7 @@ Specifies link relations. See [Web Linking](http://tools.ietf.org/html/rfc5988))
 | apps               | Lists all [applications](/docs/reference/api/apps/#application-object) that are assigned to the Group. See [Application Group Operations](/docs/reference/api/apps/#application-group-operations).          |
 | logo               | Provides links to logo images for the Group if available                     |
 | self               | The primary URL for the Group                                                                                             |
+| <ApiLifecycle access="ea" /> source             | The URL for the source [application](/docs/reference/api/apps/#application-object) of the group. This link attribute is only present in groups with `APP_GROUP` type.          |
```

Comment:
@jakubvul-okta Can we add the `<ApiLifecycle access="ea" />` tag in a table? I want to indicate this individual attribute is an early access feature.

Reply:
I found that this is in fact used in tables on other pages (though it seems to fit a bit better on other pages).

Reply:
> @jakubvul-okta Can we add the `<ApiLifecycle access="ea" />` tag in a table? I want to indicate this individual attribute is an early access feature.

We can and do add the that tag in tables, yes. And, yes, @joshreesjones-okta - sometimes formatting is problem with this label. You just have to play with, unfortunately.